src/constants/Regex.ts

```diff
@@ -2,5 +2,8 @@ export const REGEX = {
   EMAIL:
     /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,10}[a-zA-Z0-9])?)*$/g,
   ONLY_WHITESPACE: /^(?!\s*$).+/,
+  NUMBER: /^(?!0$)([1-9]\d{0,3}|1000)$/,
+  TIME: /^\d+h(?:\s+\d+m)?$/,
+  TIME_NUMBER: /^(?:[1-9]\d{0,2}|100|0)$/,
```

Comment:
^(?:[1-9]\d?|100|0)$

With that, you don't need this min max values from 0 to 100. 
101 and more will not apply.

Reply:
You are correct, but the min and max values in the TIME const aren't for validating the form, they are used in the error message for range values
https://github.com/AliSzu/Recipe-App/blob/fa27a3cafbb02feaf4d666028d068d0ea3bbbd2d/src/components/atoms/TimeField.tsx#L40-L45